magit-todos.el

```diff
@@ -853,7 +859,10 @@ sections."
                       (let* ((filename (propertize (magit-todos-item-filename item) 'face 'magit-filename))
                              (string (--> (concat indent
                                                   (when magit-todos-show-filenames
-                                                    (concat filename " "))
+                                                    (concat (if magit-todos-trim-filename-fn
```

Comment:
It's a matter of style, but I'd prefer to use the code I posted in my comment:
```el
(when magit-todos-trim-filename-fn
  (setf filename (funcall magit-todos-trim-filename-fn)))
```
It's easier to read than the `if` version.

Reply:
Done